src/_pytest/_io/pprint.py

```diff
@@ -376,7 +376,7 @@ def _pprint_simplenamespace(
         context: Set[int],
         level: int,
     ) -> None:
-        if type(object) is _types.SimpleNamespace:
+        if isinstance(object, _types.SimpleNamespace):
```

Comment:
This should stay as is if I understand the comment below correctly.

Reply:
Thank you for the reviews ! [287c651](https://github.com/pytest-dev/pytest/pull/11901/commits/287c651b7428a97777d0be4691b9a360facf8bef)

Reply:
I'm thinking let's disable this rule entirely, generally it's a good rule but pytest can do weird low-level things and we usually know what we're doing when we use `type(..) is ...`.

Reply:
All right, I'll remove the manual fix for type / is instance and rebase everything cleanly.